nuget/Microsoft.CodeMetrics/Microsoft.CodeMetrics.targets

```diff
@@ -0,0 +1,19 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <PropertyGroup>
+    <MetricsExeFolder>$(MSBuildThisFileDirectory)\..\Metrics</MetricsExeFolder>
+    <MetricsExeName>Metrics</MetricsExeName>
+    <MetricsOutputFile Condition="'$(MetricsOutputFile)' == ''">$(MSBuildProjectName).Metrics.xml</MetricsOutputFile>
```

Comment:
Is this required for legacy mode too?

Reply:
nit: maybe use `Choose-When` instead

Reply:
I'll stick with the current approach as the properties within the legacy property group just need to append `.Legacy`